settings/checkstyle/checkstyle-suppressions.xml

```diff
@@ -17,4 +17,6 @@
   <suppress checks=".*" files="[\\/]gen-java[\\/]" />
   <!-- Suppress checks in large forks to make diffing against upstream easier -->
   <suppress checks=".*" files="[\\/]grpc[\\/]ServerCallImpl.java$" />
+  <!-- Suppress visibility checks for JMH which requires public fields, etc -->
+  <suppress checks="VisibilityModifier" files="[\\/](jmh)[\\/]" />
```

Comment:
I found annotating `private` field with `@Param` just works and thus this suppression is unnecessary.